faq.html

```diff
@@ -172,6 +172,14 @@ <h1 id="how-can-i-keep-myself-safe">How much privacy does Pixelated provide?</h1
     <h1 id="sounds-good-how-can-i-contribute">Sounds good! How can I contribute?</h1>
 
     <p>We know that it can still be a bit tough to set up your Pixelated environment and find an issue that&#39;s a good fit for your skills, and we&#39;re always looking for ways to lower the entry barrier for new contributors. If you hit any roadblock, you have good ideas, or just want to share your thoughts about onboarding etc., contact our community at <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a> so we can resolve these issues for you and everyone else. We also have an <a href="irc://irc.freenode.net/pixelated">IRC channel</a>.</p>
+    <p>These principles shall guide you on your way to help build a strong ecosystem with active and enthusiastic contributors.<p>
+    <ul>
+      <li>There is no walled-off "core team". If you use or work on our code, you are welcome to the team!</li>
+      <li>Community voice is achieved after some non-trivial commits, activity in the communication channels, expression of interest, and following the Code of Conduct.</li>
+      <li>If you sign up for a feature, you claim ownership in the sense that you'll be the go-to-person.</li>
+      <li>Should you lose interest, that's ok, too. Contribution/ownership is not an obligation, and this should encourage people to accept ownership, even if for a short time.</li>
+      <li>If you accept ownership over a specific part of the code, you'll have the final say. Simple reason therefore is that dog food shall be eaten. An active contributor shall be heard, and if a contributor finds they are not using Pixelated in the real-world, they should reconsider their involvement with the project. Or, better yet, find out if there's a specific reason for it and bring it to the community's attention so that we can fix it for everyone else, too.</li>
```

Comment:
Not sure about the notion of 'ownership' here. There should be no real ownership for parts of the code. If you work more on something than others you will very likely shape it in your sense - but you don't own it. So perhaps we should rephrase this a little?